docs/src/extremevalue/generalities.md

```diff
@@ -0,0 +1,103 @@
+# [Extreme Value Copulas](@id Extreme_theory)
+
+*Extreme value copulas* are fundamental in the study of rare and extreme events due to their ability to model dependency in situations of extreme risk. An extreme value copula $C$ has the property that $$C(u_1^t, u_2^t)=(C(u_1,u_2))^t, t > 0,$$
```

Comment:
Maybe we should add a note that says that the class only contains bivariate EVCs